android/jni/tun2socks/tun2socks.c

```diff
@@ -1477,22 +1500,35 @@ int process_device_udp_packet (uint8_t *data, int data_len)
     BAddr_Print(&local_addr, local_addr_str);
     char remote_addr_str[BADDR_MAX_PRINT_LEN];
     BAddr_Print(&remote_addr, remote_addr_str);
-    BLog(BLOG_INFO, "UDP: %s -> %s. DNS: %d", local_addr_str, remote_addr_str, is_dns);
```

Comment:
We should look at this in depth when we start work to support IPv6. Create an issue until then?